src/LeftChildRightSiblingTrees.jl

```diff
@@ -163,6 +165,60 @@ function depth(node::Node, d)
     return d
 end
 
+"""
+    graftchildren!(dest, src)
+
+Move the children of `src` to become children of `dest`.
+`src` becomes a leaf node.
+"""
+function graftchildren!(dest, src)
+    for c in src
+        c.parent = dest
+    end
+    if isleaf(dest)
+        dest.child = src.child
+    else
+        lastsib = lastsibling(dest.child)
+        lastsib.sibling = src.child
+    end
+    src.child = src   # make src a leaf
+    return dest
+end
+
+"""
+    prunebranch!(node)
+
+Eliminate `node` as a child of its parent.
```

Comment:
```suggestion
Eliminate `node` from the tree.
```

Reply:
Good idea. I also mentioned "and all its children"